s2a/src/main/proto/grpc/gcp/common.proto

```diff
@@ -0,0 +1,79 @@
+// Copyright 2022 Google LLC
```

Comment:
Just to confirm, this proto will be removed from this PR once the other PR is merged?

Reply:
Makes sense, thanks for clarifying!

Reply:
Done